FueledUtils/UIExtensions.swift

```diff
@@ -114,45 +150,124 @@ public extension UIView {
 		self.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "V:|[view]|", options: [], metrics: nil, views: views))
 	}
 
-	public func snapshotImage(afterScreenUpdates: Bool = true) -> UIImage? {
+	///
+	/// Take a snapshot of the view and returns a `UIImage`.
+	///
+	/// - Parameters:
+	///   - afterScreenUpdates: Please refer to the documentation for `afterScreenUpdates` in `UIView.drawHierarchy(in:, afterScreenUpdates:)` for more info.
+	/// - Returns: The image if it could be generated. If it couldn't, for example if the `UIView`'s width or height is 0, a crash will happen at runtime.
+	///
+	/// - Note: The returns is an implicitely unwrapped optional for backward-compatibility purpose, and will be made an optional in a future release (as well as not crash)
+	///
+	public func snapshotImage(afterScreenUpdates: Bool = true) -> UIImage! {
 		return UIImage.draw(size: self.bounds.size) { _ in
 			self.drawHierarchy(in: self.bounds, afterScreenUpdates: afterScreenUpdates)
 		}
 	}
 }
 
 public extension UITableView {
+	///
+	/// Deselect given rows, optionally with an animation.
+	///
+	/// - Parameters:
+	///   - indexPaths: The rows to deselect.
+	///   - animated: `true` if you want to animate the deselection, and `false` if the change should be immediate.
+	///
+	public func deselectRows(_ indexPaths: [IndexPath], animated: Bool) {
+		for indexPath in indexPaths {
+			self.deselectRow(at: indexPath, animated: animated)
+		}
+	}
+
+	///
+	/// **Deprecated**: Please use `deselectAllRows(animated:)` instead.
+	///
+	/// Refer to the documentation for `deselectAllRows(animated:)` for more info.
+	///
+	@available(*, deprecated, renamed: "deselectAllRows(animated:)")
 	public func deselectAllRows(_ animated: Bool) {
```

Comment:
I've named the parameter `animated` for consistency with other functions, as `deselectAllRows(true)` doesn't make sense without reading its signature

Reply:
👍